### PR TITLE
Allow clearing session keys

### PR DESCRIPTION
**Pull request by KrishnaIyer**
#### Summary

Allow clearing session keys, this is needed by the migration tool.

#### Changes

- Check the changed list for `AppSKey` and `NwkSKey` and allow update.

#### Testing

🤷   let's see if this breaks anything once deployed.

### CODE REVIEW

**Review by adriansmares — APPROVED** [review]

**Comment by KrishnaIyer** [comment]
This is deployed to all TTN foundation clusters. Waiting for feedback.

**Comment by KrishnaIyer** [comment]
So this also needs `DevAddr`; https://github.com/TheThingsNetwork/lorawan-stack-migrate/blob/67b4b43c12c4ff756799cc89b3f1eb2ba6504100/pkg/source/ttnv2/source.go#L176

**Comment by KrishnaIyer** [comment]
This is deployed and tested, currently running [my dev image](https://hub.docker.com/layers/krishnaiyer/ttn/03a6c7ca/images/sha256-b9a35de3105237d1523b633f27aeaab1ba77a9f4776e3af3155a71cc7a2813a0).

**Review by adriansmares — APPROVED** [review]

File: core/handler/manager_server.go [hunk 1]
```diff
@@ -225,7 +225,8 @@ func (h *handlerManager) SetDevice(ctx context.Context, in *pb_handler.Device) (
 		changedFields := dev.ChangedFields()
 		for _, changedField := range changedFields {
 			switch changedField {
-			case "AppKey", "UsedDevNonces", "UsedAppNonces": // Allow changing the AppKey.
+			case "AppKey", "UsedDevNonces", "DevAddr", "UsedAppNonces", "NwkSKey", "AppSKey": // Allow changing the AppKey for OTAA and the session data for ABP.
+			case "FCntUp":
```
Comment (htdvisser) [inline core/handler/manager_server.go]: Is this a leftover from "just trying something" or is this intended to be a separate case?

Comment (adriansmares) [reply core/handler/manager_server.go]: This is intended:
- In v2 there are no field masks, so you need to patch the full device.
- If the device sends an uplink between a `Get` and ` Set`, the Handler detects that you're trying to update the frame counter and fails.

Do you see a better fix ?

Comment (htdvisser) [reply core/handler/manager_server.go]: Okay, fine with me, but let's add a quick comment. Can you do that?

**Review by htdvisser — APPROVED** [review]

**Pull request merged**